matminer/featurizers/site.py

```diff
@@ -481,15 +506,23 @@ def featurize(self, struct, idx):
                                             range(1, len(neigh_sites) + 1)])[0]
                         opval = opval or 0  # handles None
                         cn_fingerprint.append(wt * opval)
-
-        return cn_fingerprint
+        chem_fingerprint = []
+        if self.chem_info is not None:
+            for val in absdeltas.values():
+                chem_fingerprint.append(val / Nwt)
+        return cn_fingerprint+chem_fingerprint
```

Comment:
use Pycharm, etc. to get code formatting right (in this case, spaces surrounding the + sign)